demo_usage/competition_stereo_tutorial.ipynb

```diff
@@ -66,11 +66,11 @@
     "* **Plotly**:\n",
     "See instructions on how to install [here](https://github.com/plotly/plotly.py).\n",
     "\n",
-    "* **Disparity interpolation local module**:\n",
-    "The evaluation algorithm might need to interpolate the predicted disparity image if its density is less than 100% (please see the **Evaluating the results** cell for more details). Therefore, you will need to install the **disparity_interpolation** package using the commands below.\n",
+    "* **Disparity interpolation**:\n",
+    "The evaluation algorithm might need to interpolate the predicted disparity image if its density is less than 100% (please see the **Evaluating the results** cell for more details). Therefore, you will need to install the [numba](http://numba.pydata.org/) package for just-in-time compilation of the function `interpolate_disparity` using the command below.\n",
```

Comment:
I wonder if we should include an optional dependencies section on the main repository README. I wouldn't be opposed to adding `numba` as a required dependency -- I think we'll find other uses for it. What do you think?

Reply:
Agreed. That'd be helpful.